### DISCUSSION TITLE
Multi Currency

### DISCUSSION THREAD

**Discussion by GreenArrowlol** [Paymenter/Paymenter#264] [Ideas]
Secondary Currency + Payment Gateway customizaton. For example Paypal can be used to pay in USD and Stripe can be used to pay in EUR. We could limit them to per currencies or all currencies.

**Comment by Xteri08** [comment]
+1 this would be use full if you are doing transaction in each currency for eg
From India to India in inr it wont work but stripe or other payment method will

**Reply by GreenArrowlol** [reply]
yes my friend this feature would be soo good once added as we can separately accept usd in paypal and INR in stripe.
